src/providers/WorkflowCore.QueueProviders.SqlServer/Services/SqlConnectionHelper.cs

```diff
@@ -0,0 +1,26 @@
+﻿#region using
+
+using System;
+using System.Data.SqlClient;
+using System.Linq;
+
+#endregion
+
+namespace WorkflowCore.QueueProviders.SqlServer.Services
+{
+    public class SqlConnectionHelper
+    {
+        internal static SqlCommand CreateCommand(SqlConnection cn, SqlTransaction tx, string cmdtext, string name = null)
```

Comment:
Could we refactor this into a class that implements an interface, rather than a static method?
Maybe call it `SqlCommandExecutor` or something like that